### ISSUE TITLE
[SoC2020][idea 1] Contribute all-in-one installer for SQLFlow

### ISSUE THREAD

**Issue by lhw362950217** [sql-machine-learning/sqlflow#2260]
# 简介
[SQLFlow](https://sqlflow.org) 旨在建立数据库和 AI 引擎之间的桥梁，利用 SQL 简洁的语法和强大的描述能力降低使用 AI 的门槛。你可以从[这里](https://sql-machine-learning.github.io/doc_index/sqlflow_exapmles/)看到使用 SQLFlow 工具实现机器学习任务是多么的简单。SQLFlow 在为减轻用户负担，提升使用体验方面做了[很多工作](https://sql-machine-learning.github.io/sqlflow/doc/run/docker/)，但目前并没有达到完美的程度，我们需要更多贡献者的帮助！

# Description
[SQLFlow](https://sqlflow.org) aims to act as the bridge of DBs and AI engines. It leverage the powerful representation ability of SQL to ease the usage of AI while still keeping its simplicity. From [here](https://sql-machine-learning.github.io/doc_index/sqlflow_exapmles/) you can find out how easy it is to do machine learning task with SQLFlow. We have done [lots of work](https://sql-machine-learning.github.io/sqlflow/doc/run/docker/) to lower user's burden and promote user experience. But, as for now, it's far from perfect. We need more contributors to help us!

# 目标
现在我们希望贡献者能够开发出一个一键安装工具，帮助用户快速在 macOS/Windows/Linux 上安装 SQLFlow 工具。我们希望这个工具有如下特性：
1. 容易获取：这意味这个它是比较**轻量的，容易分发**的，我们可以将它放在 GitHub 上供用户下载 
2. 没有依赖：它是一个一键安装工具，用户下载后即可使用，而**无需手动安装**其他依赖工具
3. 容易使用：该工具应该提供**良好的用户体验**，如果出现错误应该给出清晰的提示和解决方案

# Target
Now we want our contributors to develop an all-in-one installer for SQLFlow, to help users install SQLFlow on macOs/Windows/Linux systems. We hope this tool have the following features:
1. Accessible: this means it is **light-weight**, **easy to delivery**, we can put it on GitHub for users to download
1. Self-Contains: it is an all-in-one tool, with **no extra dependencies that users have to install manually**
1. User-Friendly: it provides **good user experience**, even when it goes wrong, it should notice the user why and what to do next

# 联系我们
如果您有很棒的主意，请用下面的邮件地址联系我们。我们将会非常感谢你的参与和贡献！

邮件地址: sqlflow@list.alibaba-inc.com

# Contact us!
If you have any great idea, please contact us with following email address. We will appreciate your participation and contribution！
 
Email Address: sqlflow@list.alibaba-inc.com

**Comment by sjha2048** [comment]
Hey @lhw362950217 , I am Sahil Jha, a previous year ASoC student. 
This idea seems pretty interesting to me, also I have tried out sqlflow, how can I reach out to you for further discussion?

**Comment by lhw362950217** [comment]
> Hey @lhw362950217 , I am Sahil Jha, a previous year ASoC student.
> This idea seems pretty interesting to me, also I have tried out sqlflow, how can I reach out to you for further discussion?

@sjha2048  Thanks very much! Please contact us with this email `sqlflow@list.alibaba-inc.com`. We are expecting your excellent ideas!

**Comment by sjha2048** [comment]
Thanks @lhw362950217 , I have dropped you a mail with my ideas. 
hoping to hear from you soon.

**Comment by sjha2048** [comment]
Hi @lhw362950217 did you get my email?

**Comment by lhw362950217** [comment]
> Hi @lhw362950217 did you get my email?

Yes, I've received & replied you email. Please check your email-box.

**Comment by sjha2048** [comment]
hello @lhw362950217 , Thank you for responding, I have mailed you my draft proposal, can you please have a look and provide some feedback?

**Comment by sjha2048** [comment]
Hi @lhw362950217 , can you let me know the status of my proposal?
